src/esn.contact.libs/app/virtual/virtual-addressbook-registry.service.js

```diff
@@ -0,0 +1,30 @@
+const _ = require('lodash');
+
+(function(angular) {
```

Comment:
Would you mind removing all the IIFEs? It's a great opportunity to do it.

Reply:
the old files will be removed in another commit to prevent breaking other spa

Reply:
You need to have PRs to get the other SPAs to use `esn.contact.libs` instead and then merge all the PRs together.

Reply:
That means you need to fix the SPAs that will break because of these changes.

Reply:
First , we must delete the files and test the behavior of contact SPA , will added it in separate commit